<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Weather Widget</string>
  </property>
  <widget class="QLabel" name="lbl_day">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Понедельник</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_w1_icon">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Trash/PyQt5_Project_beta/resources/icons/project_icons/c3_r2.svg</normaloff>../../Trash/PyQt5_Project_beta/resources/icons/project_icons/c3_r2.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_w2_icon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Trash/PyQt5_Project_beta/resources/icons/project_icons/c3.svg</normaloff>../../Trash/PyQt5_Project_beta/resources/icons/project_icons/c3.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_day2">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>530</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Втр</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_w3_icon">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Trash/PyQt5_Project_beta/resources/icons/gismeteo-icons/new/c3_rs3.svg</normaloff>../../Trash/PyQt5_Project_beta/resources/icons/gismeteo-icons/new/c3_rs3.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_day3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Срд</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_w4_icon">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Trash/PyQt5_Project_beta/resources/icons/gismeteo-icons/new/d_c1_rs1.svg</normaloff>../../Trash/PyQt5_Project_beta/resources/icons/gismeteo-icons/new/d_c1_rs1.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_day4">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>530</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Чтв</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_w5_icon">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Trash/PyQt5_Project_beta/resources/icons/gismeteo-icons/new/n_c2_st.svg</normaloff>../../Trash/PyQt5_Project_beta/resources/icons/gismeteo-icons/new/n_c2_st.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_day5">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>530</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Птн</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>52</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Geliat ExtraLight</family>
     <pointsize>90</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>18:20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_time_icon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
/*background-image: url(:/newPrefix/bootstrap-icons-1.9.1/clock.svg);*/</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/newPrefix/project_icons/clock1.svg</normaloff>:/newPrefix/project_icons/clock1.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_temp_day">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>360</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Geliat ExtraLight</family>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>19°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lbl_temperature_icon">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>600</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons/icons.qrc">
     <normaloff>:/newPrefix/project_icons/thermometer-half_white.svg</normaloff>:/newPrefix/project_icons/thermometer-half_white.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/images/wtf.qrc">:/backgrounds/done_backgrounds/Main_menu_big.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_temp_day2_2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>400</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>19°C/8°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_temp_day3_2">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>400</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>19°C/8°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_temp_day4_2">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>400</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>19°C/8°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_temp_day5">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>400</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>19°C/8°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_city">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Город</string>
   </property>
  </widget>
  <zorder>lbl_background</zorder>
  <zorder>lbl_day</zorder>
  <zorder>lbl_w1_icon</zorder>
  <zorder>lbl_w2_icon</zorder>
  <zorder>lbl_day2</zorder>
  <zorder>lbl_w3_icon</zorder>
  <zorder>lbl_day3</zorder>
  <zorder>lbl_w4_icon</zorder>
  <zorder>lbl_day4</zorder>
  <zorder>lbl_w5_icon</zorder>
  <zorder>lbl_day5</zorder>
  <zorder>lbl_time</zorder>
  <zorder>lbl_time_icon</zorder>
  <zorder>lbl_temp_day</zorder>
  <zorder>lbl_temperature_icon</zorder>
  <zorder>lbl_temp_day2_2</zorder>
  <zorder>lbl_temp_day3_2</zorder>
  <zorder>lbl_temp_day4_2</zorder>
  <zorder>lbl_temp_day5</zorder>
  <zorder>lbl_city</zorder>
 </widget>
 <resources>
  <include location="../resources/images/wtf.qrc"/>
  <include location="../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
